pages/account-management.tsx

```diff
@@ -0,0 +1,139 @@
+import { useRouter } from "next/router";
+import React from "react";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import { colorMap } from "../components/Button";
+import { Input, Submit } from "../components/formComponents";
+import Layout from "../components/Layout";
+import Page, { LeftSection, RightSection } from "../components/Page";
+import { errorMessages } from "../messages";
+
+type FormValues = {
+  currentEmail: string;
+  newEmail: string;
+  password: string;
+};
+
+const AccountManagement: React.FunctionComponent = () => {
+  const intl = useIntl();
+  const { push } = useRouter();
+  const methods = useForm<FormValues>();
+  const { handleSubmit } = methods;
+  const onSubmit: SubmitHandler<FormValues> = async () => {
+    push("/passport");
+  };
+
+  return (
+    <Layout
+      title={intl.formatMessage({
+        defaultMessage: "Manage your account",
+      })}
+      headTitle={intl.formatMessage({
+        defaultMessage:
+          "Manage your account - GC Workplace Accessibility Passport",
+      })}
+      data-h2-width="b(100) m(75) l(50)"
+      data-h2-padding="b(all, none)"
+    >
+      <Page>
+        <LeftSection>
+          <FormProvider {...methods}>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <Input
+                id="currentEmail"
+                name="currentEmail"
+                type="email"
+                label={intl.formatMessage({ defaultMessage: "Current Email" })}
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+              />
+              <Input
+                id="newEmail"
+                name="newEmail"
+                type="email"
+                label={intl.formatMessage({ defaultMessage: "New email" })}
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+              />
+              <Input
+                id="password"
+                name="password"
+                type="password"
+                label={intl.formatMessage({ defaultMessage: "Password" })}
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+              />
+              <div style={{ float: "right" }}>
+                <Submit
+                  data-h2-padding="b(top-bottom, s) b(right-left, m)"
+                  data-h2-margin="b(top, s)"
+                  data-h2-font-style="b(underline)"
+                  text={intl.formatMessage({
+                    defaultMessage: "Submit email change",
+                  })}
+                />
+              </div>
+            </form>
+          </FormProvider>
+        </LeftSection>
+        <RightSection>
+          <h2 data-h2-margin="b(top, none)" data-h2-font-size="b(h3)">
+            {intl.formatMessage({ defaultMessage: "Account Tools" })}
```

Comment:
Account Tools should come in the same line. Not sure why it wraps here even though there is lot of space 
<img width="361" alt="Screen Shot 2022-05-12 at 11 58 37 AM" src="https://user-images.githubusercontent.com/89482910/168118442-073a8bde-3094-44aa-804c-d23de9f386e5.png">

Reply:
forgot to mention , there is also some text in the wireframe missed in the left section below
<img width="798" alt="Screen Shot 2022-05-12 at 1 20 23 PM" src="https://user-images.githubusercontent.com/89482910/168132803-b50870a0-516a-4e07-a786-de3115b60407.png">

Reply:
Okay I added the missing text, fixed the layout, and stopped the "Account tools" heading from breaking (i think).